### PR TITLE
Java lab15

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/tree/BinarySearchTree.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        //check if node is null, return false if so
+        if(node == null){
+            return false;
+        }
+        //check if node is equal to value
+        if(node.getValue().equals(t)){
+            return true;
+        }
+        //check the leftChild and rightChild values
+        return this.containsHelper(node.getLeftChild(), t) || this.containsHelper(node.getRightChild(), t);
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/tree/BinarySearchTree.java]: Neat!
